.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+        "@babel/preset-react"
+    ],
+    "plugins": ["@babel/plugin-syntax-jsx"]
+  }
```

Comment:
Please configure your editor to automatically add [new lines at the end of files](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#answer-729795)(tl;dr: some generally useful programs may break on files that don't have a newline at the end of the last line)

Reply:
good remark